src/Sentry/SentryOptions.cs

```diff
@@ -423,6 +423,11 @@ public double TraceSampleRate
         /// </summary>
         public ISentryTraceSampler? TraceSampler { get; set; }
 
+        /// <summary>
+        /// When set to true, Sentry captures transactions from middlewares even if the request has no legacy or Endpoint Routing features
+        /// </summary>
+        public bool TraceRequestsWithoutRouting { get; set; } = false;
```

Comment:
Sounds like this belong to `SentryAspNetCoreOptions`?

Reply:
Ohh oops